src/components/ProjectItem/Project.module.css

```diff
@@ -0,0 +1,65 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 15%;
+  padding: 1rem;
+  border-radius: 12px;
+  box-shadow: 0 2px 8px rgba(32, 28, 17, 0.26);
+  transition: all 0.3s ease-out;
+}
+
+.card:hover {
+  transform: scale(1.1);
+  box-shadow: 0 2px 8px rgba(22, 21, 21, 0.46);
+  margin: 5px;
+  background-color: #fdf0e4;
+}
```

Comment:
this needs refactoring. move all the cards to the same component.

Reply:
done